app/views/events/_event.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row">
+<% obj.each do |event| %>
```

Comment:
- [ ] Think on how to refactor this into a helper method. Avoid loops, view logic, directly in views. Having the content coming from a helper method is faster than interpreting the content inside a block in the erb file.
Check[ this article about rails mistakes, especially the part about view logic.](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make#common-mistake-2-putting-too-much-logic-in-the-view)